### PR TITLE
Log In

### PR DESCRIPTION
**Pull request by NadiaJalalTamimi**
make the login component page.

Screenshot (150)

### CODE REVIEW

**Review by modrayyes — APPROVED** [review]

File: client/src/pages/Login.jsx [hunk 1]
```diff
@@ -1,9 +1,113 @@
...
+    setValue(values);
+    sendRequest();
+  };
   return (
-    <div />
+    <div className="container">
+      <div className="loginContainer">
+        <div><img className="loginImg" src={flower} alt="flower" /></div>
+        <div className="formLogin">
+          <h1>تسجيـــل الــدخول</h1>
```
Comment (Nizar7zak) [inline client/src/pages/Login.jsx:36]: here you should use our title common component !

File: client/src/pages/Login.jsx [hunk 2]
```diff
@@ -1,9 +1,113 @@
...
+  };
+  const onFinish = (values) => {
+    setValue(values);
+    sendRequest();
+  };
   return (
-    <div />
+    <div className="container">
+      <div className="loginContainer">
+        <div><img className="loginImg" src={flower} alt="flower" /></div>
```
Comment (Nizar7zak) [inline client/src/pages/Login.jsx:34]: here you should use our Img common component,

File: client/src/pages/Login.jsx [hunk 3]
```diff
@@ -1,9 +1,113 @@
...
+            labelCol={{
+              span: 8,
+            }}
+            wrapperCol={{
+              span: 16,
+            }}
+            onFinish={onFinish}
+            autoComplete="off"
+          >
+            <Form.Item
```
Comment (Nizar7zak) [inline client/src/pages/Login.jsx:48]: you can destruct Item from Form, I think it's better if we destruct it.

**Review by Nizar7zak — CHANGES_REQUESTED** [review]

File: client/src/pages/Login.jsx [hunk 4]
```diff
@@ -1,9 +1,113 @@
...
+                </Button>
+              </div>
+
+            </Form.Item>
+
+          </Form>
+          <div className="loginLinks">
+            <a href="/signup">
+              إنشاء حساب
+            </a>
```
Comment (LinaYahya) [inline client/src/pages/Login.jsx:105]:
```suggestion
            <Link to="/signup">
              إنشاء حساب
            </Link>
```
and import link from react-router-dom

**Review by LinaYahya — CHANGES_REQUESTED** [review]

File: client/src/index.js [hunk 5]
```diff
@@ -3,11 +3,13 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import { ConfigProvider } from 'antd';
 import App from './App/index';
+import Login from './pages/Login';
 
 ReactDOM.render(
   <ConfigProvider direction="rtl">
     <React.StrictMode>
       <App />
+      <Login />
```
Comment (LinaYahya) [inline client/src/index.js:12]:
```suggestion
```

File: client/src/pages/Login.jsx [hunk 6]
```diff
@@ -1,9 +1,118 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+  Form, Input, Button,
+  message,
+} from 'antd';
+import Img from '../components/common/Img';
+// import Title from '../components/common/Title';
```
Comment (LinaYahya) [inline client/src/pages/Login.jsx:8]:
```suggestion
```
remove commented lines

File: client/src/pages/Login.jsx [hunk 7]
```diff
@@ -1,9 +1,118 @@
...
+import flower from './img/flower.png';
+import './login.css';
```
Comment (LinaYahya) [inline client/src/pages/Login.jsx:10]: make a folder for login and put your files there

File: client/src/pages/Login.jsx [hunk 8]
```diff
@@ -1,9 +1,118 @@
...
+const Login = () => {
+  const [value, setValue] = useState({ email: '', password: '' });
+  const history = useHistory();
+  const sendRequest = async () => {
+    try {
+      if (value.email && value.password.length > 6) {
+        const response = await http.post('api/v1/login', value);
+        if (response) {
+          message.success('تم تسجيل الدخول بنجاح');
+          history.push('/');
```
Comment (LinaYahya) [inline client/src/pages/Login.jsx:23]: Good Job Nadia,
we do need to setUser to user from the authProvider, so maybe we'll need to send user data from the server so we can setUser({id, name, role})

File: client/src/pages/Login.jsx [hunk 9]
```diff
@@ -1,9 +1,118 @@
...
+                <Button type="secondary" htmlType="submit" style={{ borderRadius: '5px' }}>
+                  الصفحة الرئيسية
+                </Button>
+              </div>
+
+            </Item>
+
+          </Form>
+          <div className="loginLinks">
+            <a href="/signup">
```
Comment (LinaYahya) [inline client/src/pages/Login.jsx:108]:
```suggestion
            <Link to="/signup">
```

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by Nizar7zak — APPROVED** [review]

**Pull request merged**